.github/workflows/playwright.yml

```diff
@@ -10,11 +10,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        GRAFANA_VERSION: ['latest', '9.5.5']
+        GRAFANA_VERSION: ['latest', '10.0.5', '9.5.5', '9.2.5']
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
+      - name: Copy auth file
+        run: mkdir -p packages/plugin-e2e/playwright/.auth/ && cp .github/user.json packages/plugin-e2e/playwright/.auth/user.json
```

Comment:
I wish playwright could create this file if it doesn't already exist, but doesn't seem like it's possible.